src/routes/files/handlers.js

```diff
@@ -4,6 +4,10 @@ import handleErrors from '../../lib/apiErrorHandler';
 import { File } from '../../models';
 import s3Uploader from '../../lib/s3Uploader';
 
+import ActivityReport from '../../policies/activityReport';
```

Comment:
I think calling this `ActivityReportPolicy` is a little clearer, as you do elsewhere to avoid name collisions.